go.mod

```diff
@@ -58,8 +62,13 @@ require (
 	github.com/google/gnostic-models v0.6.8 // indirect
 	github.com/google/pprof v0.0.0-20241210010833-40e02aabc2ad // indirect
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.20.0 // indirect
+	github.com/huandu/xstrings v1.5.0 // indirect
+	github.com/mitchellh/copystructure v1.2.0 // indirect
+	github.com/mitchellh/reflectwalk v1.0.2 // indirect
```

Comment:
These dependencies are even archived and not maintained anymore.

Reply:
cc @sbueringer , not sure if we should instead do the third_party approach with "minimal" change 🤔

Reply:
xref https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/3286#discussion_r1916210331